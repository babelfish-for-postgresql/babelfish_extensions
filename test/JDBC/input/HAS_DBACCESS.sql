SELECT HAS_DBACCESS('master');
GO

SELECT HAS_DBACCESS('does_not_exist');
GO

CREATE DATABASE TestDb;
GO
SELECT HAS_DBACCESS('TestDb');
GO

SELECT HAS_DBACCESS('TestDb   ');
GO

SELECT HAS_DBACCESS('   TestDb');
GO

DROP DATABASE TestDb;
GO
SELECT HAS_DBACCESS('TestDb');
GO

SELECT HAS_DBACCESS('babelfish_db');
GO
-- With COLLATION BBF_Unicode_CP1_CI_AI 
CREATE DATABASE TestDb COLLATE BBF_Unicode_CP1_CI_AI;
GO
SELECT HAS_DBACCESS('TestDb');
GO

SELECT HAS_DBACCESS('TestDb   ');
GO

SELECT HAS_DBACCESS('   TestDb');
GO

DROP DATABASE TestDb;
GO
SELECT HAS_DBACCESS('TestDb');
GO