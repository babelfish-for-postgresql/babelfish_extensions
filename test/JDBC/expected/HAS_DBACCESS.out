SELECT HAS_DBACCESS('master');
GO
~~START~~
int
1
~~END~~


SELECT HAS_DBACCESS('does_not_exist');
GO
~~START~~
int
<NULL>
~~END~~


CREATE DATABASE TestDb;
GO
SELECT HAS_DBACCESS('TestDb');
GO
~~START~~
int
1
~~END~~


SELECT HAS_DBACCESS('TestDb   ');
GO
~~START~~
int
1
~~END~~


SELECT HAS_DBACCESS('   TestDb');
GO
~~START~~
int
<NULL>
~~END~~


DROP DATABASE TestDb;
GO
SELECT HAS_DBACCESS('TestDb');
GO
~~START~~
int
<NULL>
~~END~~


SELECT HAS_DBACCESS('babelfish_db');
GO
~~START~~
int
<NULL>
~~END~~

-- With COLLATION BBF_Unicode_CP1_CI_AI 
CREATE DATABASE TestDb COLLATE BBF_Unicode_CP1_CI_AI;
GO
SELECT HAS_DBACCESS('TestDb');
GO
~~START~~
int
1
~~END~~


SELECT HAS_DBACCESS('TestDb   ');
GO
~~START~~
int
1
~~END~~


SELECT HAS_DBACCESS('   TestDb');
GO
~~START~~
int
<NULL>
~~END~~


DROP DATABASE TestDb;
GO
SELECT HAS_DBACCESS('TestDb');
GO
~~START~~
int
<NULL>
~~END~~

